query/physicalplan/aggregate.go

```diff
@@ -102,22 +102,22 @@ func chooseAggregationFunction(
 	switch aggFunc {
 	case logicalplan.AggFuncSum:
 		switch dataType.ID() {
-		case arrow.INT64:
-			return &Int64SumAggregation{}, nil
+		case arrow.INT64, arrow.FLOAT64:
```

Comment:
Since the aggregation functions now handle the type switch we should be ably to simplify this function and remove the redundant type switches here.